bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -11,7 +11,7 @@ metadata:
     olm.skipRange: '>=4.0.0 <5.0.0'
     operatorframework.io/cluster-monitoring: "true"
     operatorframework.io/suggested-namespace: openshift-windows-machine-config-operator
-    operators.operatorframework.io/builder: operator-sdk-v2.0.0+git
+    operators.operatorframework.io/builder: operator-sdk-v1.15.0+git
```

Comment:
In the commit message `Update bundle image` isn't the correct terminology here, you're updating the bundle itself. We don't maintain the bundle image, that is something that a user can build using the bundle.

Reply:
To be clear `Update bundle directory` would be more accurate